setup.py

```diff
@@ -1,7 +1,7 @@
 import setuptools
 
-with open('README.rst') as read_file:
-    long_description = read_file.read()
+#with open('README.rst') as read_file:
+#    long_description = read_file.read()
```

Comment:
Yeah doing a `pip install -e .` for the repo was giving me an error with `README.rst`, so I just commented it out to get it to install correctly.